conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```
@@ -81,7 +84,7 @@ jobs:
 {%- if azure.store_build_artifacts %}
   - script: |
         export CI=azure
-        export CI_RUN_ID=$(build.BuildNumber).$(system.JobAttempt)
+        export flow_run_id=$(build.BuildNumber).$(system.JobAttempt)
```

```suggestion
        export flow_run_id=azure_$(build.BuildNumber).$(system.JobAttempt)
```

Should this be prefixed too?